### ISSUE TITLE
This app does not stay minimize in-tray when X in the windows is clicked

### ISSUE THREAD

**Issue by leoheck** [getferdi/ferdi#719]
Using 5.5.0 (deb) on Ubuntu 20.04

This app does not stay minimize in-tray when I click the close button in the window. 

I hate that. It should minimize to tray and it should be de default behavior. There are no reason for a messaging app that closes itself when the user wants to hide it from their desktop. ALSO, when a new message comes, it should notify the user by default. 

Otherwise, why people are using a messaging app?

**Comment by k0d3r1s** [comment]
if i understand the problem correctly, this is the same problem as #657 and #697 as ferdi (in my case) is still running after pressing X but there is no icon visible and reopening ferdi would open existing session and not new one

**Comment by andrecavallari** [comment]
Also here, when I click on "x" the app closes, when I open it again I need to login again to app :(

**Comment by kytwb** [comment]
Apologies for the label, I took things a little bit too personally and reacted in a way that goes against our [code of conduct](https://github.com/getferdi/ferdi/blob/develop/CODE_OF_CONDUCT.md).

**Comment by vantezzen** [comment]
I think this is entirely personal preference: For me, when I close an application, I want it to entirely close and not run in the background.
For example, if I need to render a video in an editing software, I close all other applications so the video will render as fast as possible. I don't want Ferdi to "hide" in the tray and still consume my resources. In my case I'd use the minimize button in that case.

Please also keep in mind that keeping Ferdi active in the background is really resource-hungry: As Ferdi is a chromium-based browser it has to keep all these pages active in the background.

> when I open it again I need to login again to app
Again, as Ferdi is essentially a chromium-based browser, this depends on the service and there is nothing we can do to change that. For example, WhatsApp keeps me constantly logged in - even after restarting.

@leoheck Please keep in mind that Ferdi is an open-source application maintained by the GitHub community. If you think that this is an important feature, you can implement it into the code with an optional setting and we will be happy to merge it into our project.

**Comment by leoheck** [comment]
@vantezzen of course, I have this in mind. I also have in mind that this is a fork from Franz. Being able to add issues here does not mean users want to implement something. They can improve the app with feedback like I am doing. How many people have to see issues until someone came here to give the proper feedback? 

Well, that being said here is my humble opinion.

Message apps should not close by default. They should go to the tray instead (by default). It doesn't mean you won't be able to close the app. A simple "Close App" menu right-clicking the tray icon, and/or a menu entry in the File menu, for example, could do the same. 

The messaging app should stay always open since it is used to share information. Zero information is shared when people click on the X button to close it by mistake. If the user really wants to get rid of the annoyance of having to see notifications and messages, he can always close the app and/or disabling the notifications.

That being said, not having this by default is a path you want to follow, and this is Ok for me.

The issue here is, when I enable "Minimize to tray" in the settings, the app should not close when clicking the Window's X button. So, this is a bug. This isn't working well right now.

**Comment by vantezzen** [comment]
Of course feedback on the app is welcome and the project also profits from bug reports and feature requests.

> Message apps should not close by default

As I already established: This is personal preference - but that is why we implement settings into our app! 
In your case, it sounds like the "Minimize to tray" setting is not working as expected, making this a bug report instead of a feature request.

Looking at Ferdi's code, the problem most likely is in our method of handling the window closing:

https://github.com/getferdi/ferdi/blob/ea5a5932605894ae46cfdbc0e784eaebae380cd9/src/index.js#L248-L270

Could you please tell us your OS and Ferdi version to help investigate this bug? Also, is "Minimize Ferdi to system tray" and "Keep Ferdi in background when closing the window" enabled in your settings?

**Comment by leoheck** [comment]
Hello @vantezzen, thank you for your comprehension. This is the info you requested.

This is the Ferdi version
![image](https://user-images.githubusercontent.com/1277920/81953423-03f6b680-95de-11ea-9c90-9b93da15d52e.png)

I am running it on `Ubuntu 20.04` with the default Window Manager

This is my current general settings
![image](https://user-images.githubusercontent.com/1277920/81957522-11626f80-95e3-11ea-971e-dbf24f89b323.png)

Reviewing here right now, it seems that the close button is working hiding the window and not closing it anymore. When I launch the app again my accounts remain logged in, so that is ok.

**The issue seems to be changed a little bit. The app is just not going to the tray I expect, which is the app tray on the right side near the wifi, battery, and other status indicators.**

The app is closed (actually running in the background), but no icon is being displayed.
![image](https://user-images.githubusercontent.com/1277920/81958053-cac14500-95e3-11ea-856e-7b2f83a00848.png)

Before, when I posted this issue for the first time it was different. The app didn't run in the background when closing it in the X button (with the right setting enabled) and the tray icon was there in the right side (the Ferdi mustache)

What changed? Yesterday I have enabled the Pre-Release updates in Ubuntu and a bunch of software was upgraded. If I am not wrong, maybe something has changed the previous behavior.

**Comment by crazyquark** [comment]
I am also experiencing this on Manjaro, just the other day it was working fine now the icon no longer shows up in the tray when I close the main window.
